Solutions/NXLogAixAudit/Data Connectors/NXLogAixAudit.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "NXLogAixAudit",
+    "title": "NXLog AIX Audit",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog [AIX Audit](https://nxlog.co/documentation/nxlog-user-guide/im_aixaudit.html) data connector uses the AIX Audit subsystem to read events directly from the kernel for capturing audit events on the AIX platform. This REST API connector can efficiently export AIX Audit events to Azure Sentinel in real time.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**NXLog_parsed_AIX_Audit_view**](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/NXLogAixAudit/Parsers/NXLogAixAudit.txt) which is deployed with the Azure Sentinel Solution.",

```

Comment:
Please update the parser link as https://aka.ms/sentinel-nxlogaixaudit-parser

Reply:
Resolved with commit [f6788f0](https://github.com/Azure/Azure-Sentinel/pull/2968/commits/f6788f0d4f7f3d1543d9db4b913bbd25cefc319f)